jena-arq/src/main/java/org/apache/jena/query/Query.java

```diff
@@ -707,6 +719,8 @@ public void visit(QueryVisitor visitor)
             visitor.visitDescribeResultForm(this) ;
         if ( this.isAskType() )
             visitor.visitAskResultForm(this) ;
+        if ( this.isJsonType() )
```

Comment:
@afs added this one in order to be able to use the `Query` object in some tests...